packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html

```diff
@@ -0,0 +1,43 @@
+<div id="snippet-nested-block-quote">
+	<h3>The Gagarin Mixup</h3>
+
+	<p>Misquoting is more often than one may think. And it may happen to almost anybody. Sometimes it would be an accident, but sometimes we may consider this a conscious act. One of the better known cases is something a Soviet cosmonaut, Yuri Gagarin, was supposed to say but apparently never said. The infamous quote goes:</p>
```

Comment:
```suggestion
	<p>Misquoting is more widespread than one may think. And it may happen to almost anybody. Sometimes it would be an accident, but sometimes you may consider this a conscious act. One of the better known cases is something a Soviet cosmonaut, Yuri Gagarin, was supposed to say &mdash; but apparently never said. The infamous quote goes:</p>
```